Sources/ApolloCodegenLib/Templates/TypeTemplate.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+struct TypeTemplate {
+  let graphqlObject: GraphQLObjectType
+
+  func render() -> String {
+    TemplateString("""
+    public final class \(graphqlObject.name.firstUppercased): Object {
```

Comment:
you may want to make this `swiftName` instead of `name`

Reply:
This is to create the Swift representation of the GraphQL schema concrete types though, I believe `name` directly from the schema is what we want?

Schema type:
```
type Dog {
  name: String
}
```

Swift output:
```
public final class Dog: Object {
  ..
}
```

Reply:
ah fair. I think maybe a bunch of what I'm thinking of is dealing with pluralization of names for arrays of things, which is more of a field template issue

Reply:
Yes - all that 'fun' stuff will come after the first alpha release.

Reply:
yes fun is definitely the correct word to describe what that will be 🙃